invenio_deposit/api.py

```diff
@@ -308,3 +308,13 @@ def sort_by(*args, **kwargs):
             files_.sort_by = sort_by
 
         return files_
+
+    def get_file_by_id(self, file_id):
```

Comment:
It has been solved solved in Zenodo - https://github.com/zenodo/zenodo/blob/master/zenodo/modules/deposit/utils.py#L32-L53